client/starwhale/core/instance/view.py

```diff
@@ -43,7 +43,7 @@ def login(self, instance: str, username: str, password: str, alias: str) -> None
         r = requests.post(
             url,
             timeout=DEFAULT_HTTP_TIMEOUT,
-            data={"userName": username, "userPwd": password},
+            data=auth_request,
```

Comment:
My understanding is that login API is used to generate tokens, there is no need to send the token to login API
cc @dreamlandliu

Reply:
I hope someone API can check the token validity and get the user info.

Reply:
Maybe using `/api/v1/user/current` API to check is the better choice.

Reply:
@jialeicui  updated